src/Core/Style.js

```diff
@@ -160,39 +281,39 @@ class Style {
             this.point.radius = readVectorProperty(layer.paint['circle-radius'], zoom) || 1.5;
         } else if (layer.type === 'symbol') {
             // overlapping order
-            this.text.zOrder = readVectorProperty(layer.layout && layer.layout['symbol-z-order'] || 'auto', zoom);
+            this.text.zOrder = readVectorProperty(layer.layout && layer.layout['symbol-z-order'], zoom) || 'auto';
```

Comment:
Why there's not `this.text.zOrder` default value in constructor?

Reply:
Forgotten, thanks !